clients/hadoopfs/src/main/java/io/lakefs/LakeFSClient.java

```diff
@@ -14,24 +14,30 @@
  * This class uses the configuration to initialize API client and instance per API interface we expose.
  */
 public class LakeFSClient {
+    public static final String DEFAULT_SCHEME = "lakefs";
+    public static final String DEFAULT_ENDPOINT = "http://localhost:8000/api/v1";
     private static final String BASIC_AUTH = "basic_auth";
+
     private final ObjectsApi objects;
     private final StagingApi staging;
     private final RepositoriesApi repositories;
 
-    public LakeFSClient(Configuration conf) throws IOException {
-        String accessKey = conf.get(Constants.FS_LAKEFS_ACCESS_KEY);
+    public LakeFSClient(String scheme, Configuration conf) throws IOException {
+        String accessKey = getFSConfigurationValue(conf, scheme, "access.key");
         if (accessKey == null) {
             throw new IOException("Missing lakeFS access key");
         }
-        String secretKey = conf.get(Constants.FS_LAKEFS_SECRET_KEY);
+
+        String secretKey = getFSConfigurationValue(conf, scheme, "secret.key");
         if (secretKey == null) {
             throw new IOException("Missing lakeFS secret key");
         }
 
         ApiClient apiClient = io.lakefs.clients.api.Configuration.getDefaultApiClient();
-        String endpoint = conf.get(Constants.FS_LAKEFS_ENDPOINT_KEY, "http://localhost:8000/api/v1");
-        if (endpoint.endsWith("/")) {
+        String endpoint = getFSConfigurationValue(conf, scheme, "endpoint");
```

Comment:
Can we add an optional 3rd parameter `defaultValue`?  Then we can even handle an error if `defaultValue == null` and the variable was not set.